eclass/kernel-install.eclass

```diff
@@ -542,16 +542,15 @@ kernel-install_pkg_pretend() {
 
 	if ! use initramfs && ! has_version "${CATEGORY}/${PN}[-initramfs]"; then
 		ewarn
-		ewarn "WARNING: The standard Gentoo distribution kernel configuration"
-		ewarn "requires an initramfs to reliably boot! You have disabled the"
-		ewarn "initramfs USE flag, as a result, dracut was not pulled in as a"
-		ewarn "dependency. Please ensure that you are either overriding the standard"
-		ewarn "configuration or that an alternative initramfs generation"
-		ewarn "is installed and installkernel is configured to use it."
-		ewarn
-		ewarn "This is an advanced use case, you are on your own to ensure"
-		ewarn "that your system is bootable!"
+		ewarn "WARNING: The standard Gentoo distribution kernel configuration requires an"
+		ewarn "initramfs to reliably boot! You have disabled the initramfs USE flag, so"
+		ewarn "Dracut was not pulled in as a dependency."
+		ewarn "sys-kernel/installkernel is not configured to use an initramfs generator,"
+		ewarn "please ensure that an initramfs generator is installed, and installkernel"
+		ewarn "is configured to use it."
```

Comment:
Or users can adjust the kernel config (e.g. via `/etc/kernel/config.d`) so they don't need an initramfs

Reply:
yes, i can make that more clear, that was the intention of the last bit of the warning, like "if you don't have an initrd generator recognized by portage, you're on your own to do whatever is necessary" i feel describing building in modules may lead inexperienced users down the wrong path, so other routes should maybe be implied under the "you're on your own" bit